src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -119,17 +120,28 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
         uint8_t row;
     };
 
+    struct TabObjectEntry
+    {
+        ObjectManager::ObjectIndexId index;
+        ObjectManager::ObjectIndexEntry object;
+        bool visible;
```

Comment:
Perhaps be better to use a enum class as the `true` `false` means little.

Reply:
Just to clarify, your gripe isn't the boolean type itself, but seeing `true` and `false` used as the initialiser value elsewhere in the code?

Reply:
yes. I know we have done it in lots of places but we should try to avoid it.

Reply:
Any suggestions for naming the enum class and its members? Just `Visibility`, perhaps, given it's only used by the window? Members could be `shown` or `hidden`, perhaps?

Reply:
Or refer to the filter, e.g. `FilterStatus` with members `match` and `mismatch`? I think I prefer to keep visibility.